src/components/atoms/Text/README.md

```diff
@@ -8,4 +8,74 @@ Text for property
 
 ## Usage
 
+```tsx
+import Text from "@reearth/components/atoms/Text";
+
+
+
+<Text size="m" weight="bold"> This a Text </Text>
+<Text size="l" weight="normal" color="red"> ここはテクストです </Text>
+<Text size="l" weight="normal" isParagraph={true}> if this is a paragraph, the line height should be 1.5. and if not a praragraph, it will be te default line height 1. </Text>
```

Comment:
```suggestion
<Text size="l" weight="normal" isParagraph={true}> If you set the Text component as a paragraph, the line height will become 1.5. Otherwise, it will be the default line height of 1. </Text>
```